prod-devel-rcar.yaml

```diff
@@ -257,12 +257,32 @@ components:
       target_images:
         - "tmp/deploy/images/%{MACHINE}/Image"
         - "tmp/deploy/images/%{MACHINE}/core-image-weston-%{MACHINE}.ext4"
+
+  doma_kernel:
+    build-dir: "android_kernel"
+    sources:
+      - type: repo
+        url: https://github.com/xen-troops/android_kernel_manifest.git
+        rev: common-android12-5.4-xt-master
+        manifest: default.xml
+        depth: 1
+        groups: all
+        dir: "."
+    builder:
+      type: android_kernel
+      env:
+        - "TARGET_BOARD_PLATFORM=%{SOC_FAMILY}"
+        - "BUILD_CONFIG=common/build.config.xenvm"
+        - "SKIP_MRPROPER=1"
+      target_images:
+        - "out/android12-5.4/common/arch/arm64/boot/Image"
+
```

Comment:
Could you please add "out/android12-5.4/staging/vendor/lib/modules/pvrsrvkm.ko" as well? 
Just to be sure that Ninja will notice change in this file and trigger rebuild

Reply:
Done